dot/node_test.go

```diff
@@ -29,7 +29,17 @@ import (
 
 func DefaultTestWestendDevConfig(t *testing.T) *cfg.Config {
 	config := westenddev.DefaultConfig()
-	config.BasePath = t.TempDir()
+	var tempDir string
+	if os.Getenv("CI") == "buildjet" {
+		var err error
+		tempDir, err = os.MkdirTemp("..", "DefaultTestWestendDevConfig")
+		if err != nil {
+			t.Error(err)
+		}
+	} else {
+		tempDir = t.TempDir()
+	}
+	config.BasePath = tempDir
```

Comment:
- Changed to use `t.TempDir` as default
- Changed to use a pattern to generate a random string and append in the pattern prefix, so parallel tests don't end up using the same folder
- Changed to remove the tmp folder when the test ends (the os.MkdirTemp says `It is the caller's responsibility to remove the directory when it is no longer needed`)

```suggestion
        config.BasePath = t.TempDir()
	if os.Getenv("CI") == "buildjet" {
		config.BasePath, err = os.MkdirTemp("..", "*_wnd_dev_cfg")
		require.NoError(t, err)
		t.Cleanup(func() {
		    require.NoError(os.Remove(config.BasePath))
		})
	}
```

Reply:
I don't think we should be calling `t.TempDir` at all in this case.  Also not cleaning up the folder was an explicit choice for this CI workflow, since I think that's the problem.  Can you create another PR with your changes so we can run the workflows? I think that's the only way we would know if it fixes the issue on buildjet.

Reply:
+1 to this!